server_src/routes/databaseIssue.js

```diff
@@ -156,4 +167,14 @@ router.post("/save-connection-env", async (req, res) => {
   }
 });
 
+router.post("/restart-octofarm", async (req, res) => {
+  let serviceRestarted = false;
+  try {
+    serviceRestarted = await SystemCommands.rebootOctoFarm();
+  } catch (e) {
+    logger.error(e);
+  }
+  res.send(serviceRestarted);
+});
+
```

Comment:
Did my own file at first then thought might as well just go in here rather than adding an additional route.